docs/pages/Theming/Tokens/Basics.md

```diff
@@ -132,3 +134,16 @@ const strokeSize = {
   large: globalTokens.stroke.width40,
 };
 ```
+
+### Typography
+
+In FURN, typography alias tokens can be passed into a `Text` element or wrapped using a JSX type. For example:
+
+```tsx
+import { Body1, Text } from '@fluentui-react-native/text';
+
+const myText = <Text variant="body1">Here is some body text</Text>;
```

Comment:
Question: Is <Text variant="body1">...</Text> still okay to use, or is this discouraged? I originally put both options in, since they both seem to be valid, although the "variant prop" method will have not have Dynamic Type working properly as is.

Reply:
I'd ask PM what they'd prefer

Reply:
As a side note, this should actually be `TextV1`, not `Text`. For now I'm assuming that both are valid.